### DISCUSSION TITLE
为刻记牛杂店增添了Scoop下载方式

### DISCUSSION THREAD

**Discussion by Darkatse** [xunkong/KeqingNiuza#38] [General]
首先感谢开发者的辛勤付出，随着软件更新功能越来越完善了。为了方便某些人~~比如我自己~~使用，我为刻记牛杂店增添了Scoop下载方式，使得个人使用 更新 备份 打包更加方便了。
[Scoop](https://sspai.com/post/52496)是一个Windows下的包管理器，使得使用者可以采用命令行安装/更新/卸载软件，并且全程不需要UAC权限。特别的，Scoop 最适合安装和管理各种干净小巧的软件与各种绿色软件。并且，Scoop 也极度适合为开发者配置开发环境。

安装 scoop 后使用方法：
`scoop bucket add Darkatse https://github.com/Darkatse/Scoop-Darkatse.git`
`scoop install KeqingNiuza`

~~顺便说一下，作者可以考虑每次Release的格式统一成KeqingNiuza_x.y.z.zip的格式吗，不然的话不太好用Github Action自动更新软件源~~

**Comment by Scighost** [comment]
感谢你为本项目做出的贡献。

由于软件用到的数据提前存储在本地，每次更换卡池的时候都会进行资源更新，这种个别文件的更新不适合重新发一个Release；
项目使用Github Actions设置好了自动生成和推送，小修小补也没有发Release的必要，所以一般很久才会在Release上发一个版本。

别问为什么用启动器单独更新文件，问就是保持文件一致的最好办法。
![image](https://user-images.githubusercontent.com/61003590/139878721-b4db7eae-af59-4402-95b9-345587545015.png)
